### PR TITLE
add SimpleCHInterface kernel and test file

### PR DESCRIPTION
**Pull request by Biyss**
Refs #6910

### CODE REVIEW

**Comment by dschwen** [comment]
Please put the issue number in the _body_ and _not_ in the PR title. This is required for Github to link the two properly.

File: modules/phase_field/include/kernels/SimpleCHInterface.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+{
+public:
+  SimpleCHInterface(const InputParameters & parameters);
+
+protected:
+  virtual Real computeQpResidual();
+  virtual Real computeQpJacobian();
+
+  ///@{
+  /// Variables for second order derivatives
```
Comment (dschwen) [inline modules/phase_field/include/kernels/SimpleCHInterface.h]: Combine the two comments on one line

File: modules/phase_field/src/kernels/SimpleCHInterface.C [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    _second_phi(secondPhi()),
+    _M(getMaterialProperty<Real>("mob_name")),
+    _kappa_c(getMaterialProperty<Real>("kappa_name"))
+{
+}
+
+Real
+SimpleCHInterface::computeQpResidual()
+{
+  return   _kappa_c[_qp] * _second_u[_qp].tr() * _M[_qp] * _second_test[_i][_qp].tr();
```
Comment (dschwen) [inline modules/phase_field/src/kernels/SimpleCHInterface.C]: one space after the return. same below.

**Comment by jwpeterson** [comment]
I'm not assigned but this looks clean and fairly harmless, so...

**Pull request merged**